device_calendar/lib/src/models/platform_specifics/ios/attendee_details.dart

```diff
@@ -16,17 +17,15 @@ class IosAttendeeDetails {
     if (json == null) {
       throw ArgumentError(ErrorMessages.fromJsonMapIsNull);
     }
-    if (json['role'] != null && json['role'] is int) {
-      role = Role.values[json['role']];
-    }
+
+    role = AttendeeRole.values[json['role']];
+
     if (json['attendanceStatus'] != null && json['attendanceStatus'] is int) {
       _attendanceStatus = IosAttendanceStatus.values[json['attendanceStatus']];
     }
   }
 
   Map<String, dynamic> toJson() {
-    return <String, dynamic>{
-      'role': role?.index,
-    };
+    return <String, dynamic>{ 'role': role.index };
```

Comment:
Is role guarranteed to not be null? Might be safe to keep the role?.index just to be safe.

Reply:
It is guaranteed not to be null (always default to 'None' role), but I can add the null check in case.